src/plugins/unified_histogram/public/services/lens_vis_service.ts

```diff
@@ -499,18 +540,23 @@ export class LensVisService {
     timeRange,
     query,
     interval,
+    breakdownColumn,
   }: {
     dataView: DataView;
     timeRange: TimeRange;
     query: AggregateQuery;
     interval?: string;
+    breakdownColumn?: DatatableColumn;
   }): string => {
     const queryInterval = interval ?? computeInterval(timeRange, this.services.data);
     const language = getAggregateQueryMode(query);
     const safeQuery = removeDropCommandsFromESQLQuery(query[language]);
+    const breakdown = breakdownColumn
+      ? `, ${breakdownColumn.name} | sort ${breakdownColumn.name} asc`
```

Comment:
Shall we escape column name?

Reply:
We should, nice catch. Done here https://github.com/elastic/kibana/pull/193820/commits/9e731dfc361a48659f74ee8b3896cdc0184cbf79